safe-global/web-core
# fix: Add global focus style override

## What it solves

Resolves #1351 

## How this PR fixes it

- Adds a global style override for `focus-within` that works in webkit and firefox

## How to test it

1. Open the Safe
2. Press Tab multiple times
3. Observe all link/button elements have a focus style

## Screenshots
<img width="394" alt="Screenshot 2022-12-23 at 11 18 44" src="https://user-images.githubusercontent.com/5880855/209318336-d773a7d5-838b-4e5a-9f09-fe016d3516a8.png">
